### ISSUE TITLE
Is :preview-key gone?

### ISSUE THREAD

**Issue by ddoherty03** [minad/consult#259]
I am trying to figure out how to preview themes in `consult-theme`, but there seems to be no :preview-key keyword arguments any more, even though the README refers to it (and the variable `consult-preview-key`).

Did something on previews change recently without an update to the README?

**Comment by minad** [comment]
No nothing has been changed regarding to preview. If you do nothing and use the standard configuration, preview will be automatic.

You can configure the preview-key as described in the following sections:

* https://github.com/minad/consult#live-previews
* https://github.com/minad/consult#use-package-example
* https://github.com/minad/consult#fine-tuning-of-individual-commands

**Comment by ddoherty03** [comment]
OK.  Thanks for the quick comment.  The reason I asked is because I am trying to do selective previews of themes in `consult-theme`.  So, I added the following to my :config:
```elsip
      (setq consult-config `((consult-theme :preview-key (list ,(kbd "<S-down>") ,(kbd "<S-up>")))))
```
And it triggers this error on start-up:
```
Keyword argument :preview-key not one of (:predicate :require-match :history :default :category :initial :preview :narrow :add-history :sort :default-top :lookup)
```

**Comment by minad** [comment]
I don't see the error you are observing. This works for me:

~~~ elisp
(setq consult-config `((consult-theme :preview-key (,(kbd "<S-down>") ,(kbd "<S-up>")))))
~~~

There has been an invalid quotation in the snippet you used. You probably copied that from the readme, which also had this typo (Fixed in cf5a1845781ca57cb8b9e4af126d68d947f05127). If this does not work, please reproduce your issue with `emacs -Q`, as described in https://github.com/minad/consult/#bug-reports.

**Comment by ddoherty03** [comment]
Problem went away with your change and upgrade to newer version of consult.  Many thanks.
